public/app/plugins/datasource/prometheus/querybuilder/shared/QueryOptionGroup.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { Stack } from '@grafana/experimental';
+import { Icon, useStyles2 } from '@grafana/ui';
+import { useToggle } from 'react-use';
+
+export interface Props {
+  title: string;
+  collapsedInfo: string[];
+  children: React.ReactNode;
+}
+
+export function QueryOptionGroup({ title, children, collapsedInfo }: Props) {
```

Comment:
If this is a pattern we want to use in other Editors these might be something we want to incorporate into the EditorRow component